html/changelogs/crazylemon-cloning_brain_scan.yml

```diff
@@ -0,0 +1,4 @@
+author: Crazylemon
+delete-after: True
+changes: 
+  - rscadd: "A cloner with a tier-4 scanner will now be able to scan the brains of bodies within it, giving a chance for people who've otherwise lost their original corpse and identity."
```

Comment:
I know, but this one was in place before you gave me the notice - do you want this updated?